usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -86,7 +86,9 @@ class Command(BaseCommand):
         FROM aidlu_fabs AS aidlu LEFT JOIN raw.published_fabs AS pfabs ON (
             aidlu.transaction_unique_id = ucase(pfabs.afa_generated_unique)
         )
-        WHERE pfabs.afa_generated_unique IS NULL
+        WHERE pfabs.afa_generated_unique IS NULL AND NOT EXISTS (SELECT 1 FROM aidlu_fabs  AS aidlu LEFT JOIN raw.published_fabs AS pfabs ON (
```

Comment:
Whenever `pfabs.afa_generated_unique IS NULL` criteria is met it should be guaranteed that a record won't exist in the subquery since it relies on joining via the `pfabs.afa_generated_unique`.